README.md

```diff
@@ -19,12 +19,35 @@ pip install openshift  # openshift module for ansible
 sudo yum install jq    # jq for json processing
 ```
 
-3. **Copy source and target cluster KUBECONFIG files authenticated with  *cluster-admin* privileges to `auth` directory**
+3. A detailed inventory would be required to be provided to this ansible playbooks so it can ssh into each node in the
```

Comment:
Let's make this new step have same formatting as all other steps with a bolded title.